### DISCUSSION TITLE
In 1.2 I cannot view or edit drafts as the owner of the draft

### DISCUSSION THREAD

**Discussion by ghost** [oddvalue/laravel-drafts#46] [Q&A] [answered]
Hi,

Perhaps I am missing something. 

I am using Filament 2 with a resource route called `TicketResource`, I cannot edit or see model entries that are `is_published` = `0` aka in draft. They go to 404 unless they are published. As the creator of the draft I would be able to see and interact with.

In my `TicketResource` I have:

```
    public static function getPages(): array
    {
        return [
            'index' => Pages\ListTickets::route('/'),
            'create' => Pages\CreateTicket::route('/create'),
            'view' => Pages\ViewTicket::route('/{record}'),
            'edit' => Pages\EditTicket::route('/{record}/edit'),
            'activities' => Pages\ListTicketActivities::route('/{record}/activities'),
        ];
    }
```

Can you assist with what I may be missing?

**Comment by oddvalue** [comment, marked as answer]
@jeremy-at-evosus you'll need to disable the global scope in your resource query.

I believe adding this to your resource will do the trick:

```php
public function getEloquentQuery()
{
    return parent::getEloquentQuery()->withDrafts();
}
```
